features/net/network-socket/Socket.h

```diff
@@ -182,6 +182,8 @@ class Socket {
         attach(mbed::callback(obj, method));
     }
 
+    NetworkStack *stack() const { return _stack; }
```

Comment:
Why is this function exposed publicly in the `Socket` class? The internal `NetworkStack` is intentionally hidden from the user API.